diff --git a/cmd/lego/main.go b/cmd/lego/main.go
index 6a80fbf..8f01bc8 100644
--- a/cmd/lego/main.go
+++ b/cmd/lego/main.go
@@ -13,2 +13,3 @@ import (
 	"github.com/xenolf/lego/log"
+	"github.com/xenolf/lego/platform/config/env"
 )
@@ -36,3 +37,4 @@ func main() {
 	}
-	app.Flags = cmd.CreateFlags(defaultPath)
+	legoPath := env.GetOrDefaultString("LEGOPATH", defaultPath)
+	app.Flags = cmd.CreateFlags(legoPath)
 
