diff --git a/cmd/lego/main.go b/cmd/lego/main.go
index 30ac914..042e90d 100644
--- a/cmd/lego/main.go
+++ b/cmd/lego/main.go
@@ -12,2 +12,3 @@ import (
 	"github.com/go-acme/lego/v3/log"
+	"github.com/go-acme/lego/v3/platform/config/env"
 	"github.com/urfave/cli"
@@ -36,3 +37,4 @@ func main() {
 	}
-	app.Flags = cmd.CreateFlags(defaultPath)
+	legoPath := env.GetOrDefaultString("LEGOPATH", defaultPath)
+	app.Flags = cmd.CreateFlags(legoPath)
 
